.remarkrc

```diff
@@ -0,0 +1,6 @@
+{
+  "plugins": [
```

Comment:
https://github.com/remarkjs/remark-lint#presets
Can also enforce the markdown default, however, this is rather strict & uses default lint-length of 80 among other things.

We can update rulesets to expand, however, with modern monitors, I'm not fond of such low values. This is potentially an important accessibility issue & I'm happy for it to be called out as such. The linting auto-fix doesn't support this & reworking the dozens of findings seems wasteful. Consistency brings required compatibility up to scratch. With addition of this file, we can tweak as we see fit